### PR TITLE
HTML CSS_Homework_Gayathri

### PR DESCRIPTION
**Pull request by GayathriVenkatraman**

### CODE REVIEW

File: html-css/week1/styles.css
Comment (Khandohii) [inline html-css/week1/styles.css:1]: Please, try to keep your code clean and aligned, without unnecessary line breaks for better readability and scalability. 
The extension "Prettier" for VSCode helps a lot with this 👍

**Review by Khandohii — CHANGES_REQUESTED** [review]
Well done @GayathriVenkatraman 👍 
Left a few comments 😊

File: html-css/week1/index.html [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    </section>
+    <section id="projects" class="projects_section">
+        <h2>Portfolio Section</h2>
+        <div class="project_description">
+            <h3>Survey Form</h3>
+            <p><strong>Description: &nbsp;</strong></p>
+                <p>A responsive survey form created as part of the FreeCodeCamp curriculum, designed to capture user feedback in a structured format.
+                <br>This project showcases my understanding of HTML structure, CSS styling, and form handling to create an accessible, user-friendly design.</p>
+                <p><strong>Technologies Used: &nbsp;</strong> HTML, CSS</p>
+                <button type="button"><a href="https://www.freecodecamp.org/learn/2022/responsive-web-design/build-a-survey-form-project/build-a-survey-form">View FCC Survey Form</a></button>
```
[the remark this answers is not in the document]
Comment (GayathriVenkatraman) [reply html-css/week1/index.html:84]: Okay

File: html-css/week1/styles.css
Comment (GayathriVenkatraman) [reply html-css/week1/styles.css:1]: Thanks, this is really useful

File: html-css/week1/styles.css [hunk 2]
```diff
@@ -0,0 +1,911 @@
...
+    padding: 0;
+}
+
+body{
+    background-color: #FFDFD3;
+    
+}
+
+header{
+    width: 2020px;
```
[the remark this answers is not in the document]
Comment (GayathriVenkatraman) [reply html-css/week1/styles.css:12]: Done

File: html-css/week1/styles.css
[the remark this answers is not in the document]
Comment (GayathriVenkatraman) [reply html-css/week1/styles.css:1]: media queries added

[the remark this answers is not in the document]
Comment (Khandohii) [reply html-css/week1/styles.css:1]: I can't see commits with last updates in this PR.

[the remark this answers is not in the document]
Comment (GayathriVenkatraman) [reply html-css/week1/styles.css:1]: I have been working on these comments in my local repository. I will update them here as well.

**Pull request merged**